src/bind.c

```diff
@@ -7,63 +9,51 @@ static int bind_one(sqlite3_stmt *stmt, int n, struct value *value)
 	int rc;
 
 	switch (value->type) {
-	case SQLITE_INTEGER:
-		rc = sqlite3_bind_int64(stmt, n, value->integer);
-		break;
-	case SQLITE_FLOAT:
-		rc = sqlite3_bind_double(stmt, n, value->real);
-		break;
-	case SQLITE_BLOB:
-		rc = sqlite3_bind_blob(stmt, n, value->blob.base,
-						(int)value->blob.len,
-						SQLITE_STATIC);
-		break;
-	case SQLITE_NULL:
-		rc = sqlite3_bind_null(stmt, n);
-		break;
-	case SQLITE_TEXT:
-		rc = sqlite3_bind_text(stmt, n, value->text, -1,
-						SQLITE_STATIC);
-		break;
-	case DQLITE_ISO8601:
-		rc = sqlite3_bind_text(stmt, n, value->text, -1,
-						SQLITE_STATIC);
-		break;
-	case DQLITE_BOOLEAN:
-		rc = sqlite3_bind_int64(stmt, n,
-					value->boolean == 0 ? 0 : 1);
-		break;
-	default:
-		rc = DQLITE_PROTO;
-		break;
+		case SQLITE_INTEGER:
+			rc = sqlite3_bind_int64(stmt, n, value->integer);
+			break;
+		case SQLITE_FLOAT:
+			rc = sqlite3_bind_double(stmt, n, value->real);
+			break;
+		case SQLITE_BLOB:
+			rc = sqlite3_bind_blob(stmt, n, value->blob.base,
+					       (int)value->blob.len,
+					       SQLITE_STATIC);
+			break;
+		case SQLITE_NULL:
+			rc = sqlite3_bind_null(stmt, n);
+			break;
+		case SQLITE_TEXT:
+			rc = sqlite3_bind_text(stmt, n, value->text, -1,
+					       SQLITE_STATIC);
+			break;
+		case DQLITE_ISO8601:
+			rc = sqlite3_bind_text(stmt, n, value->text, -1,
+					       SQLITE_STATIC);
+			break;
+		case DQLITE_BOOLEAN:
+			rc = sqlite3_bind_int64(stmt, n,
+						value->boolean == 0 ? 0 : 1);
+			break;
+		default:
+			rc = SQLITE_ERROR;
+			break;
```

Comment:
Looks like this hunk is mostly around changing white-spaces. If this is true, can we return the initial version back?